src/clarity-angular/layout/nav/header.spec.ts

```diff
@@ -11,21 +11,23 @@ import {ClrIconModule} from "../../icon/icon.module";
 import {ClrNavigationModule} from "./navigation.module";
 
 @Component({
-    template: `
-        <clr-header class="header">
-            <div class="branding">
-                <a href="#" class="nav-link">
-                    <clr-icon shape="vm-bug"></clr-icon>
-                    <span class="title">Title</span>
-                </a>
-            </div>
-            <div class="header-nav" [clr-nav-level]="1">
-                <a class="active nav-link" href="javascript://"><span class="nav-text">Components</span></a>
-            </div>
-            <div class="header-nav" [clr-nav-level]="2">
-                <a class="active nav-link" href="javascript://"><span class="nav-text">About</span></a>
-            </div>
-        </clr-header>
+    template: `        
+        <clr-main-container>
```

Comment:
@youdz: I had to wrap the `clr-header` with `clr-main-container` in the tests. Is there an alternative to this?